docs/content/howto/open-any-file.md

```diff
@@ -3,9 +3,9 @@ title: Open any file
 order: -10
 ---
 
-The Rerun Viewer has built-in support for opening many kinds of files, and can be extended to open any other file type without needing to modify the Rerun codebase itself.
+The Rerun Viewer and SDK have built-in support for opening many kinds of files, and can be extended to support any other file type without needing to modify the Rerun codebase itself.
 
-The viewer can load files in 3 different ways:
+The Viewer can load files in 3 different ways:
 - via CLI arguments (e.g. `rerun myfile.jpeg`),
 - using drag-and-drop,
 - using the open dialog in the Rerun Viewer.
```

Comment:
```suggestion
- via a direct log call (see below)
```

Reply:
The Viewer can't do log calls :thinking:

Reply:
hm right this is a list of the viewer's ability. Okay nevermind